bunq/response_data.go

```diff
@@ -63,6 +63,14 @@ type responseMasterCardActionGet struct {
 	Pagination Pagination `json:"Pagination"`
 }
 
+// ResponseSchedulePaymentGet The schedule payment response object.
+type ResponseSchedulePaymentGet struct {
+	Response []struct {
+		SchedulePayment SchedulePayment `json:"ScheduledPayment"`
```

Comment:
Oh no, the endpoint is `schedule-payment` but the object is called `ScheduledPayment` with the `d`? Shall we just name all the structs `ScheduledPayment` with the `d` in this SDK?

Reply:
Done for the schedule related stuff